OHHTTPStubs/Sources/OHHTTPStubs.m

```diff
@@ -443,6 +444,8 @@ - (void)streamDataForClient:(id<NSURLProtocolClient>)client
             timingInfo.chunkSizePerSlot = ((stubResponse.dataSize/stubResponse.responseTime) * timingInfo.slotTime);
         }
         
+        timingInfo.bodyOffset = stubResponse.bodyOffset;
```

Comment:
I'm not sure I like that hack.

1. `bodyOffset` is not really related to some "timing info" and the pace at which the data is sent periodically, so it's not logical to add it to this struct
2. This modifies the library core just to be able to support Mocktail files

If I add support for Mocktail, given that it's not a feature used by many, I'll probably add it as a opt-in submodule in my podspec, so that people don't get useless extra code if they don't use it, and people needing it simply have to explicitly request `OHHTTPStubs/Mocktail` in their Podfile to get the extra feature. This modification does not allow us to do that given that it modifies the core of OHHTTPStubs.